tasmota/xsns_69_opentherm_protocol.ino

```diff
@@ -552,7 +587,7 @@ void sns_opentherm_dump_telemetry()
     for (int i = 0; i < SNS_OT_COMMANDS_COUNT; ++i)
     {
         struct OpenThermCommandT *cmd = &sns_opentherm_commands[i];
-        if (!cmd->m_flags.supported)
+        if (!cmd->m_flags.supported || cmd->m_flags.skip)
```

Comment:
Are you sure you want to skip it? You're probably interested in the result of the BLOR call and it's available only through Telemetry.

Reply:
Thank you. It's bug.

 I will test code and push a new version.  (Remote update boiler controller is not a good idea)

Reply:
now it return 
"BLOR":{"lo":255,"hi":0} - it's OK